test/integration/issues_test.lua

```diff
@@ -111,18 +113,34 @@ function g.test_broken_replica()
     ]])
 
     local function issue_fmt(from, to)
-        return {
-            level = 'critical',
-            topic = 'replication',
-            replicaset_uuid = g.cluster:server(to).replicaset_uuid,
-            instance_uuid = g.cluster:server(to).instance_uuid,
+        local message
+        if tarantool_version > '2.8' then
```

Comment:
This is a time bomb:

```
tarantool> "2.8.1" > "2.8"
---
- true
...

tarantool> "2.9" > "2.8"
---
- true
...

tarantool> "2.10" > "2.8"
---
- false
...
```

Also, should it be `>=`?

Reply:
defused